src/backend/expungeservice/crawler/crawler.py

```diff
@@ -35,7 +35,9 @@ def login(self, username, password, close_session=False) -> bool:
 
         return Crawler.__login_validation(self.response)
 
-    def search(self, first_name, last_name, middle_name="", birth_date="") -> List[AmbiguousCase]:
+    def search(
+        self, first_name, last_name, middle_name="", birth_date=""
+    ) -> Tuple[List[AmbiguousCase], List[Question]]:
```

Comment:
To support interactive updates as the user submits one or more answers, the search endpoint response needs to also provide the Answers that they have submitted so far. In the first response from the search endpoint, there won't be any answers attached but the response structure is the same. Up to you if you think that's next-PR material.

Reply:
The frontend will continue to hit the disambiguate endpoint which returns the questions (with answers) in the record summary object.

Reply:
oh right!